kore/src/main/scala/org/kframework/POSet.scala

```diff
@@ -1,56 +1,61 @@
 // Copyright (c) K Team. All Rights Reserved.
 package org.kframework
 
-import org.kframework.utils.errorsystem.KEMException
-
-import java.util
 import collection._
+import java.util
+import org.kframework.utils.errorsystem.KEMException
 import scala.annotation.tailrec
 
 /**
  * A partially ordered set based on an initial set of direct relations.
  */
 class POSet[T](val directRelations: Set[(T, T)]) extends Serializable {
   // convert the input set of relations to Map form for performance
-  private val directRelationsMap: Map[T, Set[T]] = directRelations groupBy { _._1 } mapValues { _ map { _._2 } toSet } map identity
+  private val directRelationsMap: Map[T, Set[T]] =
+    directRelations.groupBy(_._1).mapValues(_.map(_._2) toSet).map(identity)
 
   lazy val elements: Set[T] = directRelations.flatMap(a => Set(a._1, a._2))
 
-  lazy val sortedElements: scala.collection.immutable.List[T] = TopologicalSort.tsort(directRelations).toList
+  lazy val sortedElements: scala.collection.immutable.List[T] =
+    TopologicalSort.tsort(directRelations).toList
 
   /**
-   * Internal private method. Computes the transitive closer of the initial relations.
-   * It also checks for cycles during construction and throws an exception if it finds any.
+   * Internal private method. Computes the transitive closer of the initial relations. It also
+   * checks for cycles during construction and throws an exception if it finds any.
    *
-   * The implementation is simple. It links each element to the successors of its successors.
-   * TODO: there may be a more efficient algorithm (low priority)
+   * The implementation is simple. It links each element to the successors of its successors. TODO:
+   * there may be a more efficient algorithm (low priority)
```

Comment:
This doesn't look right. Can you add a new line so it doesn't move the TODO at the end?